pkg/certify/external-bash/driver-call.go

```diff
@@ -0,0 +1,337 @@
+package externalBash
+
+import (
+	"bytes"
+	"encoding/json"
+	"flag"
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/pkg/errors"
+	"github.com/wongma7/csi-certify/pkg/certify/utils"
+	"k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/kubernetes/pkg/api/legacyscheme"
+	"k8s.io/kubernetes/test/e2e/framework"
+	"k8s.io/kubernetes/test/e2e/storage/testpatterns"
+	"k8s.io/kubernetes/test/e2e/storage/testsuites"
+	"os/exec"
+	"strings"
+)
+
+const (
+	getDriverInfo = "getDriverInfo"
+	createVolume  = "createVolume"
+)
+
+var RunCustomTestDriver = true
+var scriptName = ""
```

Comment:
this could also be a field of bashDriver I think

Reply:
I missed this comment. RunCustomTestDriver is used outside of this file, [here](https://github.com/wongma7/csi-certify/blob/ext-testdriver-nfs/pkg/certify/certify.go#L23).

Reply:
but can put scriptName in bashDriver.

Reply:
actually not sure about scriptName, since it is used in checkBashFuncExists.